pom.xml

```diff
@@ -43,6 +43,26 @@
 			<version>1.1.10</version>
 		</dependency>
 
+		<!-- Guava -->
+		<dependency>
+			<groupId>com.google.guava</groupId>
+			<artifactId>guava</artifactId>
+			<version>23.6-jre</version>
+		</dependency>
+
+		<!-- JMH Benchmark -->

```

Comment:
Since the benchmark libs are only used for testing, their dependencies should be moved to the testing block and they should have `<scope>test</scope>`